handsontable/test/e2e/helpers/dom/element.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('DOM helpers', () => {
+  describe('observeVisibilityChangeOnce', () => {
+    it('should recognize when an element becomes visible (once) and trigger the provided callback when that happens', async() => {
+      const observeVisibilityChangeOnce = Handsontable.dom.observeVisibilityChangeOnce;
+      const element = document.createElement('DIV');
```

Comment:
```suggestion
      const element = document.createElement('div');
```
Creating elements with capital letters reminds me of the old-school JS style. I'd change that :)